src/main/java/nl/haarlem/translations/zdstozgw/config/ConfigService.java

```diff
@@ -76,6 +77,13 @@ private void validateConfiguration() throws Exception {
 		log.debug("heeftAlsGemachtigde:" + rolomschrijving.getHeeftAlsGemachtigde());
 		log.debug("heeftAlsOverigeBetrokkene:" + rolomschrijving.getHeeftAlsOverigBetrokkene());
 
+		this.configuration.getBeeindigZaakWanneerEinddatum().size();
```

Comment:
Is this line necessary? and is it null safe?

Reply:
created an issue #259 for this point